ui/app/services/watch-list.js

```diff
@@ -1,11 +1,16 @@
-import { computed } from '@ember/object';
+import { computed, set } from '@ember/object';
 import { readOnly } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 import { copy } from 'ember-copy';
 import Service from '@ember/service';
 
 let list = {};
 
 export default class WatchListService extends Service {
+  @service store;
+
+  jobUpdateCount = 0;
```

Comment:
@philrenaud I tried to make the minimal surface change here. We could try to make this more generic by converting this into a dictionary, however, I imagine we could gut the watcher system once its time for a redesign.